ecl/dns/v2/_proxy.py

```diff
@@ -124,13 +124,20 @@ def get_name_server(self, zone_id):
         """
         return list(self._list(name_server.NameServer, zone_id=zone_id))
 
-    def recordsets(self, zone_id):
+    def recordsets(self, zone_id, limit, marker):
         """
         This lists all recordsets in a zone.
         :param zone_id: ID for the zone
+        :param limit:Requests a page size of items. Returns a number of items up to a limit value. Use the limit parameter to make an initial limited request and use the ID of the last-seen item from the response as the marker parameter value in a subsequent limited request.
```

Comment:
1. We had better to write the range of limit(1-500) here.
2. This is so picky, but you don't have a space after colon.
```suggestion
        :param limit: Requests a page size of items. Returns a number of items up to a limit value. Use the limit parameter to make an initial limited request and use the ID of the last-seen item from the response as the marker parameter value in a subsequent limited request.
```

Reply:
1. I think it is correct.However, there is a reason for doing so.
   The docstring of the other method describes the Parameter description of the API reference.
2. I understand.

Reply:
It is not needed docstring is the same with API reference. We had better to write it because it is important information.